### ISSUE TITLE
Star messages as favorites and have somewhere to see all starred messages

### ISSUE THREAD

**Issue by marceloschmidt** [RocketChat/Rocket.Chat#310]

**Comment by marceloschmidt** [comment]
Adding another icon for starring a message (and later on pinning) will break our layout.

This is ok:
![image](https://cloud.githubusercontent.com/assets/1986378/9895098/f85f0c5c-5c01-11e5-8a8f-6c7cc78c3eb6.png)

But this is not:
![image](https://cloud.githubusercontent.com/assets/1986378/9895102/09b72e08-5c02-11e5-8dd0-9a87d8366a97.png)

What can we do? Add only one icon with dropdown? We haven't decided if we are going with a specific dropdown package yet, have we?

cc: @RocketChat/owners

**Comment by geekgonecrazy** [comment]
yeah we've almost got to get a method for the drop down figured out :(  No way we can fit more then one icon over there

**Comment by engelgabriel** [comment]
I hate drop down.. but I can't think of a better solution :(

**Comment by geekgonecrazy** [comment]
What if dropdown desktop and use the popover on long press that's in place for reporting?

**Comment by engelgabriel** [comment]
Agreed.. that's the best option I can think of.

On the mobile.. the long hold it the only option really... on the desktop.. I wish I could think of something better.

**Comment by marceloschmidt** [comment]
Depends on #828

**Comment by gmsecrieru** [comment]
How is this going so far? I believe this is a very requested feature and I have time to help if needed :D

**Comment by geekgonecrazy** [comment]
@gmsecrieru I resynced the branch recently.  But @marceloschmidt has done most of it.

**Comment by gmsecrieru** [comment]
Closing as this was already merged into `master`.
